main.py

```diff
@@ -13,9 +15,45 @@
                                              initial_distribution=v,
                                              num_stages=N, stopping_time=tau).create()
 
-tree.bulls_eye_plot(dot_size=5, radius=300)
+# tree.bulls_eye_plot(dot_size=5, radius=300)
+#
+# tree.set_data_at_node(4, {"a": 1})
+# print(sum(tree.probability_of_node(tree.nodes_at_stage(8))))
+#
+# print(tree)
+
+# RAOCP generation -----------------------------------------------------------------------------------------------------
+
+root_state = np.array([[1],
+                       [1]])  # n vector
+
+Aw1 = np.eye(2)
+Aw2 = 2*np.eye(2)
+Aw3 = 3*np.eye(2)
+As = [Aw1, Aw2, Aw3]  # n x n matrices
+
+Bw1 = np.eye(2)
+Bw2 = 2*np.eye(2)
+Bw3 = 3*np.eye(2)
+Bs = [Bw1, Bw2, Bw3]  # n x u matrices
+
+cost_type = "quadratic"
+Q = 10*np.eye(2)  # n x n matrix
+R = np.eye(2)  # u x u matrix OR scalar
+Pf = 5*np.eye(2)  # n x n matrix
+
+(risk_type, alpha) = ("AVAR", 0.5)
+E = np.eye(2)  # p x n matrix (mu is in R^n)
+F = np.eye(2)  # p x r matrix
+cone = "Rn+"
```

Comment:
`cone` should rather be an instance of a class `Cone`

Reply:
wip